### DISCUSSION TITLE
Visualization of differential ChromVAR motifs

### DISCUSSION THREAD

**Discussion by sneddonucsf** [GreenleafLab/ArchR#1279] [Questions / Documentation] [answered]
Hello, 

I am trying to compare the ChromVAR deviations of two clusters to get differential deviations. I am doing this through: 
```
diffMotif <- getMarkerFeatures(
  ArchRProj = proj, 
  testMethod = "wilcoxon",
  useGroups = "Clust1",
  bgdGroups = "Clust2",
  binarize = FALSE,
  useMatrix = "MotifMatrix",
  groupBy = "predictedGroup_Un",
  useSeqnames="z")
```

I would like to visualize the results in some way. Which of the parameters would I use for the cutoffs (FDR? From my understanding, logFC is meaningless here) and how would I go about visualizing the results?

**Comment by wuv21** [comment, marked as answer]
From my previous post with Ryan (#1177), he suggested the usage of meandiff to observe the direction of change. I used FDR for my cutoff. You can extract the FDR and meandiff values from your diffMotif variable (which is a SummarizedExperiment object). The motif names are also located in that object.

**Reply by dgagler** [reply]
Hey,

I'm a bit confused by how to interface with the resulting SummarizedExperiment object. It appears it's not possible to use getMarkers() like with other matrices, which is unfortunate. In lieu of this, I'm poking around the object and I've been able to access things like the MeanDiff via 
```
varDev.markers@assays@data@listData[["MeanDiff"]]
```
There's no sign of what the motifs are, though. You mention that is it in the object. Can you specify where?
